showroom.js

```diff
@@ -0,0 +1,69 @@
+//Problem : Create a application for a showroom showing 3 types of car having some common data names and some uncommon data names. Use inheritance to accomplish it.
+var Vehicle = function(data) {
+  this.name = data.name;
+  this.model = data.model;
+  this.make = data.make;
+  this.color = data.color;
+};
+
+var SUV = function(data) {
+  Vehicle.call(this, data);
+  this.airbag = data.airbag;
+  this.foglight = data.airbag;
+  this.type = "SUV";
+  this.avg = "11";
+};
+
+SUV.prototype = Vehicle.prototype;
```

Comment:
@abansal26 Is prototype property necessary to include here? If yes, Please clarify.

Reply:
The assignment to do the program using prototype and prototypical inheritance. So i used prototype.